subprojects/gradle/runner/service/src/main/kotlin/com/avito/runner/service/worker/device/adb/AdbDevice.kt

```diff
@@ -38,9 +38,19 @@ data class AdbDevice(
 ) : Device {
 
     override val api: Int by lazy {
```

Comment:
Could we take API from the configuration? And try to load API Property only to check that our API and API from the emulator are the same

Reply:
Added to MBS-8531